dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferMethods.cs

```diff
@@ -19,14 +19,7 @@ public class TransferMethods
                 FilePath = $"{localPath}\\{keyName}",
             });
 
-            if (File.Exists($"{localPath}\\{keyName}"))
-            {
-                return true;
-            }
-            else
-            {
-                return false;
-            }
+            return (File.Exists($"{localPath}\\{keyName}"));
```

Comment:
I think the methods and class in this file should have comment blocks.

Reply:
Added.